docs/operations/runbooks/downtime_incident_management.md

```diff
@@ -0,0 +1,31 @@
+# Downtime Incident Management Runbook
+
+ Our team has agreed upon steps for handling incidents that cause our site to go offline or become unusable for users. For this document, an incident refers to one in which manage.get.gov is offline or displaying error 400/500 HTTP errors on all pages. However, for this document to apply the cause of the problem must be a critical bug in our code or one of our providers having an outage, not to be confused with a cyber security incident. This document should not be used in response to any type of cyber security incident.
+
+## Response management rules
+
+The following set of rules should be followed while an incident is in progress.
+
+- The person who first notices that the site is down is responsible for using @here and notifying in #dotgov-announce that production is down
```

Comment:
fullstops after every bullet point.

Reply:
what? Chaotically switching between having periods and not having periods doesn't make sense? 

Will change this